--- origsrc/openjpeg-version.2.1/src/lib/openjp3d/CMakeLists.txt	2014-04-29 07:58:10.000000000 -0500
+++ src/openjpeg-version.2.1/src/lib/openjp3d/CMakeLists.txt	2015-10-27 12:44:25.898884600 -0500
@@ -31,10 +31,12 @@ set_target_properties(${OPENJP3D_LIBRARY
 # Install library
 install(TARGETS ${OPENJP3D_LIBRARY_NAME}
   EXPORT OpenJP3DTargets
-  DESTINATION ${OPENJPEG_INSTALL_LIB_DIR}
-  COMPONENT Libraries
+  RUNTIME DESTINATION ${OPENJPEG_INSTALL_BIN_DIR} COMPONENT Applications
+  LIBRARY DESTINATION ${OPENJPEG_INSTALL_LIB_DIR} COMPONENT Libraries
+  ARCHIVE DESTINATION ${OPENJPEG_INSTALL_LIB_DIR} COMPONENT Libraries
 )
 
+
 # Install includes files
 install(FILES openjp3d.h
   DESTINATION ${OPENJPEG_INSTALL_INCLUDE_DIR}
--- origsrc/openjpeg-version.2.1/src/lib/openmj2/CMakeLists.txt	2014-04-29 07:58:10.000000000 -0500
+++ src/openjpeg-version.2.1/src/lib/openmj2/CMakeLists.txt	2015-10-27 12:45:14.148511500 -0500
@@ -50,8 +50,9 @@ set_target_properties(${OPENMJ2_LIBRARY_
 # Install library
 install(TARGETS ${OPENMJ2_LIBRARY_NAME}
   EXPORT OpenMJ2Targets
-  DESTINATION ${OPENJPEG_INSTALL_LIB_DIR}
-  COMPONENT Libraries
+  RUNTIME DESTINATION ${OPENJPEG_INSTALL_BIN_DIR} COMPONENT Applications
+  LIBRARY DESTINATION ${OPENJPEG_INSTALL_LIB_DIR} COMPONENT Libraries
+  ARCHIVE DESTINATION ${OPENJPEG_INSTALL_LIB_DIR} COMPONENT Libraries
 )
 
 # Install includes files
